code/modules/mob/living/carbon/xenomorph/castes/carrier/abilities_carrier.dm

```diff
@@ -394,8 +394,11 @@ GLOBAL_LIST_INIT(hugger_images_list,  list(
 	if(!can_use_ability(A))
 		return fail_activate()
 
+	///Finding embryo in the target
 	var/obj/item/alien_embryo/young = locate() in victim
+	///How long the target will be under debuff. Depends on the stage of the embryo
 	var/debuff = young.stage + 1
+	///How much stamina damage will be done. Depending on stage of embryo and carrier maturity.
```

Comment:
autodoc is not needed for vars inside a proc. However you can of course have comments if you think they're needed, but that's generally on the same line, and usually only needed if something isn't entirely clear at first glance.